modules/wallet/defrag.go

```diff
@@ -130,9 +129,16 @@ func (w *Wallet) threadedDefragWallet() {
 		w.mu.Unlock()
 		return
 	}
+	w.mu.Unlock()
+
+	// get the required fee for the defrag transaction
+	// 35 outputs at an estimated 250 bytes needed per output means about a 10kb
+	// total transaction.
+	fee := w.defragFee(10000)
```

Comment:
Does this even merit a helper function anymore? I think this code can just be inlined. Comment also needs to be formatted. Also, it's probably worth doing the full multiplication here. E.g. numOutputs * sizeAvgOutput instead of hardcoding 10,000.

Ideally the transaction builder itself could figure out how many bytes are needed to sign a transaction, but that's more complexity than we need at the moment.